packages/shopify-codemod/transforms/iife-to-ternary-expression.js

```diff
@@ -0,0 +1,48 @@
+export default function iifeToTernaryExpression({source}, {jscodeshift: j}, {printOptions = {}}) {
+  function isImmediatelyInvokedFunctionExpression(expressionPath) {
```

Comment:
I'm all for not using annoying contractions in method names.  It's commendable that you typed all of this out!

That said, `iife` is used in the line immediately above this, and it's a generally accepted abbreviation, so feel free to use it here, too.